test/NumberTheory/QuadFormAndIsom.jl

```diff
@@ -251,6 +251,11 @@ end
   r = enumerate_classes_of_lattices_with_isometry(L, 30; char_poly=(x-1)^2*cyclotomic(30, x))
   @test length(r) == 1
   @test det(invariant_lattice(r[1])) == -1
+  
+  r_local = enumerate_classes_of_lattices_with_isometry(L,2;char_poly=(x-1)^4*(x+1)^6,fix_root=5,_local=true)
+  r_global = enumerate_classes_of_lattices_with_isometry(L,2;char_poly=(x-1)^4*(x+1)^6,fix_root=5,_local=false)
```

Comment:
```suggestion
  r_local = enumerate_classes_of_lattices_with_isometry(L, 2; char_poly=(x-1)^4*(x+1)^6, _local=true)
  r_global = enumerate_classes_of_lattices_with_isometry(L, 2; char_poly=(x-1)^4*(x+1)^6, _local=false)
```
The `fix_root` are unnecessary here.